website/routes.py

```diff
@@ -337,6 +339,14 @@ def make_url_map(app):
             json_renderer,
         )
     ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/confirmed_emails/', 'get', auth_views.verified_email_get, json_renderer),
```

Comment:
@sloria, I use currently verified_email_get for testing.  Is there a better way to get the list of users to add in tests?